src/components/node/Node.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactNode } from 'react';
+import clsx from 'clsx';
+
+export interface NodeData {
+  body?: ReactNode;
+  links: NodeLink[];
+}
+
+interface NodeLink {
+  title: string;
+  node: NodeData;
+}
+
+export default function Node({ data }: { data: NodeData }) {
```

Comment:
This is a terrible name for this "component" and I think it is meaningless to have a component like this because the idea passing "node data" is just terrible. If you really want a re-usable component, made a "Tabs" component which switch between displaying multiple panels of content and use saner syntax like:

 ```jsx
<Tabs id="TabsExample" onChange={this.handleTabChange} selectedTabId="code">
    <Tab id="code" title="Code" panel={<CodePanel />} />
    <Tab id="translations" title="Translations" panel={<TranslationsPanel />} />
    <Tab id="other" title="Other" panel={<OtherPanel />} />
</Tabs>
```

Reply:
I made a new pr here: https://github.com/jellyfin/jellyfin.org/pull/907

See if this option looks better